.github/workflows/reusable-build-test-release.yml

```diff
@@ -1592,6 +1595,183 @@ jobs:
           path: "${{ needs.setup.outputs.directory-path }}/test-results/*.xml"
           reporter: java-junit
 
+  run-escu-tests:
+    if: ${{ needs.test-inventory.outputs.escu == 'true' && ( github.base_ref == 'main' || github.ref_name == 'main' || github.base_ref == 'develop' || github.ref_name == 'develop' ) }}
+    needs:
+      - build
+      - test-inventory
+      - setup
+      - meta
+    runs-on: ubuntu-18.04
```

Comment:
Please run on latest, I believe ubuntu-18.04 is not supported anymore.

Reply:
Updated image tag to the latest